server/uploader.js

```diff
@@ -81,84 +61,130 @@ module.exports = {
    */
   bigwigtoBcwig: function(prefix, bwFile, bigWigWoWigAddr) {
```

Comment:
Please try to camelcase variables and function names. Abbreviations are uppercase by default. I changed this to "bigWigToBCWig".

Besides, "bigWigWoWigAddr" seems to be a replacement mistake. I've corrected it.